src/layout/List/types.d.ts

```diff
@@ -12,6 +12,7 @@ export interface ILayoutCompList extends ILayoutCompBase<'List'> {
   dataListId: string;
   secure?: boolean;
   bindingToShowInSummary?: string;
+  chosenPrioritizedColumsMobile?: string[];
```

Comment:
Remember to add extra properties to the layout json schema as well. :smiley:

Reply:
Also, I find the words "chosen" and "prioritized" to be a bit vague when referring to a specific feature. Are these the columns you _choose_ to display on mobile, **or** are they _prioritized_ in a way? From glancing at the code, it seems this property needs to be set, and have to include all column keys you want to display on mobile...

Could we instead make it so that all columns are shown on mobile if this property is not set? And then possibly call it something like `tableHeadersMobile`? That way it's obvious it's the same functionality as the existing `tableHeaders` functionality, just for mobile. Alternatively `mobileColumns`, as that indicates it's the same as `sortableColumns`, also just for mobile.